src/tests/end-to-end/test_simple-test.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+import tempfile as tpf
+from distutils.dir_util import copy_tree
+
+import subprocess
+
+import numpy as np
+
+def find_integrity_path(output_dir):
+    op = []
+    for path in Path(output_dir).rglob('checkIntegrity.txt'):
+        op.append(path)
+    assert len(op) == 1
+    return op[0]
+
+def check_integrity(output : Path, expected_values : np.array):
+    assert len(expected_values) == 8
+    integrity_path = find_integrity_path(output)
+    integrity_file = open(str(integrity_path), 'r')
+    output_values = list()
+    for x in integrity_file:
+        output_values.append(float(x))
+    np.testing.assert_equal(expected_values, output_values)
```

Comment:
to ease the reading I propose that this support function returns the output values and that the assert is performed in the test function

```suggestion
def get_interity_check_values(output : Path) -> np.array :
    integrity_path = find_integrity_path(output)
    integrity_file = open(str(integrity_path), 'r')
    output_values = list()
    for x in integrity_file:
        output_values.append(float(x))
    assert len(output_values) == 8
    return output_values
```